hooks/pre_commit/20-certs_update.rb

```diff
@@ -0,0 +1,14 @@
+if module_enabled?('certs')
+  if app_value(:certs_update_server_ca) && !module_enabled?('katello')
+    fail_and_exit("--certs-update-server-ca needs to be used with katello", 101)
+  end
+
+  ca_file   = param('certs', 'server_ca_cert').value
+  cert_file = param('certs', 'server_cert').value
+  key_file  = param('certs', 'server_key').value
+
+  if !app_value(:certs_skip_check) && !cert_file.to_s.empty? &&
+     (app_value(:certs_update_server_ca) || app_value(:certs_update_server))
```

Comment:
Not needed in this PR, but I'd still consider always running `katello-certs-check` if there are custom certs.

Reply:
What is the best way to detect custom certs?

I suppose `param('certs', 'server_cert').nil? || param('certs', 'server_key).nil? || param('certs', 'server_ca_cert').nil?`

Reply:
It already has `cert_file.to_s.empty?`, which should suffice. After this is merged, I'd open a new PR that effectively drops this line.